gramps_webapi/api/resources/family.py

```diff
@@ -0,0 +1,53 @@
+"""Family API resource."""
+
+import gramps.gen.filters.rules.family as rule_classes
+
+from .base import (GrampsObjectProtectedResource, GrampsObjectResourceHelper,
+                   GrampsObjectsProtectedResource)
+from .filter import apply_filter_rules, list_filter_rules
+from .util import (get_children_for_references, get_events_for_references,
+                   get_family_profile_for_object, get_media_for_references,
+                   get_person_by_handle)
+
+
+class FamilyResourceHelper(GrampsObjectResourceHelper):
+    """Family resource helper."""
+
+    gramps_class_name = "Family"
+
+    def object_extend(self, obj):  # pylint: disable=no-self-use
```

Comment:
There is self use in this method (so the comment should go away). This was probably done erroneously by myself.

Reply:
Cleaned up